### DISCUSSION TITLE
<title>Não consigo de forma alguma verificar o 2FA no meu GITHUB

### DISCUSSION THREAD

**Discussion by RafaelAndradeDEV** [community/community#83096] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Isso devido não haver suporte para o SMS no Brazil e o aplicativo de TOTF para gerar código nunca gera um código correto, já fiz scan do qrcode várias vezes, assim como também a escrita da key, mas nada deu certo. Como devo proceder?

**Comment by m-mourouh** [comment]
**Double-check the Configuration:**

Make sure that you have entered the correct phone number with the correct country code.
Verify that the time on your device is set correctly, as TOTP (Time-based One-Time Password) codes are time-sensitive.
Alternative Authentication Methods:

If SMS is not working, check if there are alternative authentication methods available. Some services provide backup options, such as email verification or authenticator apps.
### Use Authenticator Apps:

Instead of relying on SMS, consider using authenticator apps like Google Authenticator, Authy, or similar apps. These apps generate time-based codes that can be used for two-factor authentication.
Check for Service Outages:

Sometimes, issues can be on the service provider's end. Check if there are any service outages or reported issues with the two-factor authentication system.
### Contact Support:

Reach out to the support team of the service you are trying to access. They may be able to assist you in resolving the issue or provide alternative methods of authentication.
### Update the App:

Ensure that the TOTP application you are using is up-to-date. Developers often release updates to fix bugs and improve compatibility.
### Clear App Data/Cache:

If the TOTP application has a cache or stored data, try clearing it. Sometimes, cached data can cause issues with code generation.
Check for Phone Number Format:

Confirm that you are entering the phone number in the correct format, including the country code. Different services may have specific requirements for how phone numbers should be entered.
### Try Another Device:

If possible, try setting up two-factor authentication on another device to see if the issue persists. This can help identify whether the problem is device-specific.
### Consider Support Alternatives:

If the standard two-factor authentication methods are not working, some services offer alternative authentication methods or support for special cases. Check the service's documentation or contact their support for assistance.

**Reply by jrgarou** [reply]
There's no Brazil in the list of country calling codes:
![Screen Shot 2024-01-18 at 12 14 48](https://github.com/community/community/assets/320157/4f6d9362-fef8-4f53-9dc8-c4db21624aaa)

**Comment by ghostinhershell** [comment]
Hi @RafaelAndradeDEV, 

Thanks so much for reaching out in the Community! At the moment Brazil isn't a supported country for SMS authentication but we do update this list periodically.

You mentioned that your TOTP app isn't reliably sending you codes that allow you to login to GitHub. I recommend checking the accuracy of the system clock against a known time standard. TOTP stands for Time-based One-Time Password, and requires access to an accurate clock in order for the codes to be generated correctly. In other words, your system time must be accurate and match the timezone that's set in your TOTP app to work.
